### PR TITLE
Additional alerts for Prometheus - part 1

### PR DESCRIPTION
**Pull request by rafzei**
Second PR regarding alerting rules...
This PR contains:
- Additional alerting rules: container, kafka, node, postgresql, prometheus
- How-to update
- Changelog update

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/prometheus/files/prometheus/rules/node.rules [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    expr: sum by (instance) (irate(node_disk_written_bytes_total[2m])) / 1024 / 1024 > 50
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Unusual disk write rate (instance {{ $labels.instance }})"
+      description: "Disk is probably writing too much data (> 50 MB/s)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+
+  - alert: OutOfDiskSpace
+    expr: (node_filesystem_free_bytes  / node_filesystem_size_bytes * 100 ) < 10  and (node_filesystem_free_bytes/1024/1024/1024  < 1.5)
```
Comment (plirglo) [inline core/src/epicli/data/common/ansible/playbooks/roles/prometheus/files/prometheus/rules/node.rules]: space

File: core/src/epicli/data/common/ansible/playbooks/roles/prometheus/files/prometheus/rules/node.rules [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Out of disk space (instance {{ $labels.instance }})"
+      description: "Disk is almost full (< 10% left)\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+
+  - alert: OutOfInodes
+    expr: node_filesystem_files_free/ node_filesystem_files * 100 < 10
```
Comment (plirglo) [inline core/src/epicli/data/common/ansible/playbooks/roles/prometheus/files/prometheus/rules/node.rules]:
```suggestion
    expr: node_filesystem_files_free / node_filesystem_files * 100 < 10
```

File: core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml [hunk 3]
```diff
@@ -8,3 +8,66 @@
     mode: u=rw,go=
   when: specification.alert_rules.common | bool
   notify: Reload Prometheus
+
+- name: Copy Prometheus alerting rules
+  copy:
+    src: "prometheus/rules/prometheus.rules"
```
Comment (plirglo) [inline core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml]: choose whether with or without quotation marks - i think we don't need "" in this case

File: core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml [hunk 4]
```diff
@@ -8,3 +8,66 @@
...
+    validate: /usr/local/bin/promtool check rules %s
+    owner: prometheus
+    group: prometheus
+    mode: u=rw,go=
+  when: specification.alert_rules.prometheus | bool
+  notify: Reload Prometheus
+
+- name: Copy node alerting rules
+  copy:
+    src: "prometheus/rules/node.rules"
```
Comment (plirglo) [inline core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml]: ""

File: core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml [hunk 5]
```diff
@@ -8,3 +8,66 @@
...
+    group: prometheus
+    mode: u=rw,go=
+  when:
+  - groups.kubernetes_node[0] is defined
+  - specification.alert_rules.node | bool
+  notify: Reload Prometheus
+
+- name: Copy container alerting rules
+  copy:
+    src: "prometheus/rules/container.rules"
```
Comment (plirglo) [inline core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/provide-rules.yml]: ""

**Review by plirglo — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Pull request merged**